wit/deps/sockets/ip-name-lookup.wit

```diff
@@ -1,6 +1,6 @@
 
 interface ip-name-lookup {
-	use wasi:poll/poll.{pollable}
+	use wasi:io/poll.{pollable}
```

Comment:
I think somewhere the upgraded deps are out of sync - this one already has the merge of `poll` into the `wasi:io` package, but that isnt reflected in the rest of this tree.

Reply:
Should be fixed by now.